skimage/feature/_canny.py

```diff
@@ -11,11 +11,13 @@
 All rights reserved.
 Original author: Lee Kamentsky
 """
+from __future__ import division, print_function, absolute_import
```

Comment:
This forces Python 2 into Python 3 behavior regarding true division, the print function, and import statements. 

The core Python packages have placed this in every executable file, because it saves contributors on 2.x significant time debugging. Also, the division change can fly under the radar and create problems.